travis/install.sh

```diff
@@ -64,14 +64,41 @@ setup_travis()
 
 setup_linux()
 {
-    sudo apt-get install -y software-properties-common openssh-server libncurses5-dev libncursesw5-dev openjdk-8-jre-headless
-    RELEASE="$(lsb-release -sr)"
-    if [[ "$RELEASE" < "16.04" ]]; then
-        sudo apt-add-repository --yes ppa:pwntools/binutils
-        sudo apt-get update
-        sudo apt-get install -y binutils-arm-linux-gnu binutils-mips-linux-gnu binutils-powerpc-linux-gnu
+    declare -A osInfo;
```

Comment:
This isn't necessary as we are not testing on any OS except Ubuntu.

This is not a set-up or install script, it is used EXCLUSIVELY to set up the environment to run tests.

Reply:
Wouldn't the ability for testing for other platforms (for the cost of just having 10+ lines of setup script) be a good thing? At least the possibility for it, not even asking for automated, hosted, CI etc.
Because for me, even when I force my system to use py2, not all tests seem to pass, which indicates that pwntools don't fully function on arch...

Reply:
Have you looked at `TESTING.md`?  I strongly recommend using the supplied dockerfile for testing.

Reply:
I actually agree to include the augmented setup script, since I personally use many different distros for testing pwntools. While it may seem hard to maintain, it could be awesome for the contributors to be able to easily test stuff locally.

Since this script already checks if the machine runs under linux or osx, we can go on and check the distro.